docs/user-tutorial/benchmarks/model-benchmarks.md

```diff
@@ -42,3 +42,24 @@ For inference, supported percentiles include
 | model-benchmarks/pytorch-${model_name}/${precision}_inference_throughput                | throughput (samples/s) | The average inference throughput with fp32/fp16 precision.                   |
```

Comment:
Whether we need to change training throughput? It should be training throughput per GPU?